step/step.go

```diff
@@ -138,7 +138,7 @@ func updateVersionNumbersInInfoPlist(helper *projectmanager.ProjectHelper, targe
 	absoluteInfoPlistPath := filepath.Join(filepath.Dir(helper.XcProj.Path), infoPlistPath)
 
 	infoPlist, format, _ := xcodeproj.ReadPlistFile(absoluteInfoPlistPath)
-	infoPlist["CFBundleVersion"] = bundleVersion
+	infoPlist["CFBundleVersion"] = strconv.Itoa(bundleVersion)
```

Comment:
Hey, my builds are currently failing on CI. by bundle version is 7.20.0 which I believe would fail `strconv.Itoa`. What was the need for this change? Cheers

Reply:
./prog.go:11:27: cannot use "7.20.0" (untyped string constant) as int value in argument to strconv.Itoa

Reply:
Same here. What was the need for this update?

Reply:
While I agree that [Apple considers CFBundleVersion to be in format X.Y.Z](https://developer.apple.com/documentation/bundleresources/information_property_list/cfbundleversion), in Bitrise docs [there are multiple mentions](https://devcenter.bitrise.io/en/builds/build-numbering-and-app-versioning.html) that build numbers should be plain integers and not in format X.Y.Z.

Unless I'm mistaken, adopting this step in Bitrise implies to convert your CFBundleVersion to a plain integer.